ipn/ipnlocal/local.go

```diff
@@ -428,6 +446,15 @@ func NewLocalBackend(logf logger.Logf, logID logid.PublicID, sys *tsd.System, lo
 	captiveCtx, captiveCancel := context.WithCancel(ctx)
 	captiveCancel()
 
+	m := metrics{
+		advertisedRoutes: sys.UserMetricsRegistry().NewGauge(
+			"tailscaled_advertised_routes", "Number of advertised network routes (e.g. by a subnet router)"),
+		approvedRoutes: sys.UserMetricsRegistry().NewGauge(
+			"tailscaled_approved_routes", "Number of approved network routes (e.g. by a subnet router)"),
+		primaryRoutes: sys.UserMetricsRegistry().NewGauge(
+			"tailscaled_primary_routes", "Number of network routes for which this node is a primary router (in high availability configuration)"),
```

Comment:
Does this include routes for which it's the only router? E.g. let's say a node is a subnet router for two routes, A and B. Route A is also routed by some other nodes but this node is the current primary. This node is the only node that is advertising route B. Would this count be 1 or 2?

Reply:
yes, I'll try to reword it

Reply:
"Number of network routes for which this node is the primary router, in a high availability configuration, this indicates the amount of routes served through this router" maybe?

Reply:
Maybe "Number of network routes for which this node is the primary or sole router"?